cmd/witnessctl/api.go

```diff
@@ -105,9 +104,6 @@ func (d Device) getLogMessages(cmd byte) (string, error) {
 			}
 			w.Write(rsp.GetPayload())
 			req.Continue = true
-
-			// Don't overload the HID endpoint
-			time.Sleep(20 * time.Millisecond)
```

Comment:
Is this change related to this PR? The description does not mention it

Reply:
This isn't necessary any more (this was a previous attempt to address the stability issues)